Header.jsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { View, Text, StyleSheet, Button } from "react-native";
+
+export const Headers = () => (
+  <View style={styles.container}>
+    <Text style={styles.left}>Cancel</Text>
+    <Text style={styles.main}> New task </Text>
+    <Text style={styles.right}>Save</Text>
+  </View>
+);
+
+const styles = StyleSheet.create({
+  container: {
+    height: 70,
+    paddingTop: 30,
+    backgroundColor: "rgb(31, 134, 255)",
```

Comment:
Try to be consistent with the color format. We have `rgb` and **hex** (e.g `#FFFF`), and we should stick to only one. Probably you will see `rgba` for simplicity when you are trying to change the level of `opacity` of the desired color. But every color can be made with **HEX**, and in RN we usually stick to it.

Here's the reference if you want to know mow about how colors work in React Native: https://reactnative.dev/docs/colors

And here's a color picker: https://www.w3schools.com/colors/colors_picker.asp

Can you figure out which is the associated **hex** value to the `rgb` you use in this line?

```suggestion
    backgroundColor: "#.....",
```